core/email.py

```diff
@@ -209,39 +185,35 @@ def email_to_admin(
         username=None,
         user_email=None,
         cc_user=True,
-        admin_user=None,
         request_tracker=False,
         html=False):
     """
     Send a basic email to the admins. Nothing more than subject and message
```

Comment:
Just a point of commentary. This function's behavior is so needlessly complex. All this if/else bs would be eliminated with a simpler signature:
```
def email_to_admin(subject, body, from, cc=None, html=False)
```